django/db/models/options.py

```diff
@@ -354,6 +336,21 @@ def swapped(self):
                     return swapped_for
         return None
 
+    @property
+    def managers(self):
+        managers = []
+        bases = (b for b in self.model.mro() if hasattr(b, '_meta'))
+        for depth, base in enumerate(bases):
+            for manager in base._meta.local_managers:
+                manager = copy.copy(manager)
```

Comment:
it's weird to me that we have to copy managers both here and in `ManagerDescriptor`. why couldn't the descriptor find the appropriate manager from `model._meta.managers` instead of re-copying it at runtime? Would require tracking managers by name in `Options`, but that doesn't seem like a big deal.